From f7f9bfe5883b688f5a5c942a8d4ecfac1605f474 Mon Sep 17 00:00:00 2001
From: Mark Brown <broonie@linaro.org>
Date: Tue, 17 Jun 2014 12:39:05 +0100
Subject: [PATCH] configs: Enable some SELinux options

Going forward Android will require SELinux and we may as well make sure
it does't get in the way for other configurations.

Signed-off-by: Mark Brown <broonie@linaro.org>
---
 linaro/configs/linaro-base.conf | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/linaro/configs/linaro-base.conf b/linaro/configs/linaro-base.conf
index fa38b9e..1cdf0ba 100644
--- a/linaro/configs/linaro-base.conf
+++ b/linaro/configs/linaro-base.conf
@@ -103,3 +103,13 @@ CONFIG_FTRACE_SYSCALLS=y
 CONFIG_STACK_TRACER=y
 CONFIG_FUNCTION_PROFILER=y
 CONFIG_MAILBOX=y
+CONFIG_AUDIT=y
+CONFIG_NF_CONNTRACK_SECMARK=y
+CONFIG_NETFILTER_XT_TARGET_CONNSECMARK=y
+CONFIG_NETFILTER_XT_TARGET_SECMARK=y
+CONFIG_IP_NF_SECURITY=y
+CONFIG_SECURITY=y
+CONFIG_SECURITY_NETWORK=y
+CONFIG_LSM_MMAP_MIN_ADDR=4096
+CONFIG_SECURITY_SELINUX=y
+CONFIG_EXT4_FS_SECURITY=y
-- 
1.8.0

